recipes/tf2onnx/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "tf2onnx" %}
+{% set version = "1.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/onnx/tensorflow-onnx/archive/v{{ version }}.tar.gz
+  sha256: 567f11c963c91d6f719c9ff2ac23a430d65fe037b4c3aacc30eaf12b7d60861c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true     # [win or py2k]
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+    - tensorflow >=1.5.0
```

Comment:
You shouldn't need any host dependencies other than python, pip and pytest-runner

Reply:
yeap, I know, that was just for testing purposes.

But thanks for the follow-up! :)